lib/primitiveToGeometry.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+var readAccessor = require('./readAccessor');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+var modeToPrimitiveType = require('./modeToPrimitiveType');
+var Cesium = require('cesium');
+var Geometry = Cesium.Geometry;
+var GeometryAttribute = Cesium.GeometryAttribute;
+var ComponentDatatype = Cesium.ComponentDatatype;
+var Cartesian3 = Cesium.Cartesian3;
+var defined = Cesium.defined;
+
+module.exports = primitiveToGeometry;
+
+function primitiveToGeometry(gltf, primitive) {
```

Comment:
Given that these are all fundamental graphics terms, throughout, I think we should be explicit about what is glTF and what is Cesium, e.g., `gltfPrimitiveToCesiumGeometry`.